conf/defaults.ini

```diff
@@ -501,11 +501,6 @@ nodata_or_nullvalues = no_data
 # This limit will protect the server from render overloading and make sure notifications are sent out quickly
 concurrent_render_limit = 5
 
-#################################### Explore #############################
-[explore]
-# Enable the Explore section
-enabled = false
```

Comment:
Could there be a circumstance where someone would not want Explore? What's the limit on datasource access, org? Team? Could someone have used the Editor role extensively and might not want to give access to Explore?

Reply:
Maybe. Should we just switch the default setting?
I would like to remove it unless we have a case for disabling it since I'd like to have as few settings as possible.